### PR TITLE
Data localization suite tile

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/data-localization/key-management/geo-key-manager.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+pcx_content_type: concept
+title: Geo Key Manager
+weight: 3
+---
+
+# Geo Key Manager
+
+By default, private keys will be encrypted and securely distributed to each data center, where they can be utilized for local SSL/TLS termination. Geo Key Manager allows you to choose where you want to store your private keys. Geo Key Manager was restricted to the US, EU, and high-security data centers, but with the new version of Geo Key Manager, available in Closed Beta, we allow you to create allowlists and blocklists of countries in which your private keys will be stored. That means that you will be able to create specifications, like "store my private keys in Australia only" or "store my private keys in the EU and in the UK".
```
Comment (tlozoot) [inline content/data-localization/key-management/geo-key-manager.md]: Nitpick, but the tone of this paragraph is somewhat "blogpost" or "sales-y". It would be good to just neutrally document the differences between GKM v1 vs v2 and explain how to use each one.

File: content/data-localization/regional-services/get-started.md [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+layout: list
+---
+
+## Get Started
+
+You can use Regional Services through the dashboard or via API.
+
+### Regional Services in the dash
+
+To use regional services, you need to first create a DNS record in the dashboard:
```
Comment (tlozoot) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
To use Regional Services, you need to first create a DNS record in the dashboard:
```

File: content/data-localization/regional-services/get-started.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+pcx_content_type: get-started
+weight: 1
+layout: list
+---
+
+## Get Started
+
+You can use Regional Services through the dashboard or via API.
+
+### Regional Services in the dash
```
Comment (tlozoot) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
### Regional Services in the dashboard
```

**Review by tlozoot — APPROVED** [review]
Don't think I've ever been this excited by a documentation update. Go go go go!

**Comment by tlozoot** [comment]
@Achiel Suggests using something other than the ❌ emoji. I guess we could just say "not available" or even use ✘

File: content/data-localization/metadata-boundary/get-started.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+---
+title: Get started
+pcx_content_type: get-started
+weight: 1
+---
+
+# Get Started
+
+You can configure the Customer Metadata Boundary and change the region where your logs and analytics are stored via API.
```
Comment (soheiokamoto) [inline content/data-localization/metadata-boundary/get-started.md]: You could link to API docs around https://developers.cloudflare.com/api/operations/get-accounts-account_identifier-logs-control-cmb-config

I could only get a link to each endpoint, but not at the top of the section. Maybe we could make a feature request for it.

File: content/data-localization/_index.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+weight: 1
+meta:
+  title: Data Localization Suite
+---
+
+# Data Localization Suite
+
+The Data Localization Suite (DLS) is a set of products that helps customers who want to maintain local control over their traffic while retaining the security benefits of a global network.
+
+DLS is currently available for customers on the Enterprise plan. Please reach out to your Customer Success Manager for more information about purchasing DLS.
```
Comment (marciocloudflare) [inline content/data-localization/_index.md]:
```suggestion
DLS is currently available for customers on the Enterprise plan. Reach out to your Customer Success Manager for more information about purchasing DLS.
```

File: content/data-localization/_index.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+The Data Localization Suite consists of the following products:
```
Comment (marciocloudflare) [inline content/data-localization/_index.md:15]:
```suggestion

The Data Localization Suite consists of the following products:
```

File: content/data-localization/_index.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+- [Key Management](/data-localization/key-management/)
+- [Regional Services](/data-localization/regional-services/)
+- [Customer Metadata Boundary](/data-localization/metadata-boundary/)
+
+Support by product and region is summarized in the following table:
+
+| Region | Geo Key Manager | Regional Services | Customer Metadata Boundary |
```
Comment (marciocloudflare) [inline content/data-localization/_index.md:23]: Maybe `Geo Key Manager` could link to `Key Management > Geo Key Manager`? Or be replaced by `Key Management`? Not sure. However, it's the only column that doesn't reflect the menu, and this might cause some confusion as to what is the Geo Key Manager bc we don't talk about it before showing its availability.

File: content/data-localization/_index.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+| India | &#x2705; | &#x2705; | ✘ |
+
+Overview by product-behavior is summarized in the following table. Below you can find the table legend to help you read the table:
+
+✅ Product works with no caveats <br>
+🚧 Product can be used with some caveats <br>
+✘ Product cannot be used <br>
+⚫️ Not applicable
+
+| Suite/Category | Product | Geo Key Manager | Regional Services | Customer Metadata Boundary |
```
Comment (marciocloudflare) [inline content/data-localization/_index.md:40]: Same comment here regarding Geo Key Manager

File: content/data-localization/_index.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+| --- | --- | --- | --- | --- |
+| Application Performance | Caching/CDN | ✅ | ✅ | ✅ |
+| | DNS | ⚫️ | ⚫️ | 🚧[^1] | 
+| | Image Resizing | ✅ | ✅ | 🚧[^1] |
+| | Load Balancing | ✅ | ✅ | 🚧[^1] |
+| | Stream Delivery | ✅ | ✅ | ✅ |
+| | Tiered Caching | ✅ | 🚧[^2] | 🚧[^2] |
+| | Waiting Room | ⚫️ | ✅ | 🚧[^1] |
+| | Zaraz | ✅ | ✅ | 🚧[^1] |
+| Application Security | Advanced Certificates Manager | ⚫️ | ⚫️ | ⚫️ |
```
Comment (marciocloudflare) [inline content/data-localization/_index.md]: The blog talks about `Advanced Certificate Manager` not `Certificates`.

File: content/data-localization/_index.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+[^6]:Cannot be used with Customer Metadata Boundary outside of US region.
+[^7]:[Jurisdiction restrictions for Durable Objects](/workers/runtime-apis/durable-objects/#restricting-objects-to-a-jurisdiction).
+[^8]:You can not yet specify region location for object storage; this is expected in 2023.
+[^9]:Argo cannot be used with Regional Services.
+[^10]:Argo cannot be used with Customer Metadata Boundary.
+[^11]:BYOIP cannot be used with Regional Services.
+[^12]:Logpull not available when using Customer Metadata Boundary outside US region. Logs may be stored and retrieved with [Logs Engine](https://blog.cloudflare.com/announcing-logs-engine/) which is adding region support in 2023.
+[^13]:Logpush available with Customer Metadata Boundary for HTTP requests and Firewall events. Please contact your Customer Success Manager if you need to push another dataset.
+[^14]:Access App SSL keys can use Geo Key Manager. [Access JWT](/cloudflare-one/identity/authorization-cookie/validating-json/) is not yet localized.
+[^15]:Can be localized to US FedRAMP region only; More regions coming in 2023.
```
Comment (marciocloudflare) [inline content/data-localization/_index.md]:
```suggestion
[^15]:Can be localized to US FedRAMP region only. More regions coming in 2023.
```

File: content/data-localization/key-management/_index.md [hunk 11]
```diff
@@ -0,0 +1,9 @@
+---
+pcx_content_type: concept
+title: Key Management
+weight: 1
+---
+
+# Key Management
+
+Cloudflare offers [Keyless SSL](https://www.cloudflare.com/ssl/keyless-ssl/) and [Geo Key Manager](/ssl/edge-certificates/custom-certificates#geo-key-manager-private-key-restriction), which ensures that private SSL/TLS key material stays in your intended region. Keyless SSL ensures that Cloudflare never has possession of the private key material at all; Geo Key Manager uses Keyless SSL to ensure the keys never leave the specified region.
```
Comment (marciocloudflare) [inline content/data-localization/key-management/_index.md]: Any specific reason we link to the Keyless SSL product page instead of the docs (`/ssl/keyless-ssl/`)?

File: content/data-localization/key-management/geo-key-manager.md [hunk 12]
```diff
@@ -0,0 +1,11 @@
+---
+pcx_content_type: concept
+title: Geo Key Manager
+weight: 3
+---
+
+# Geo Key Manager
+
+By default, private keys will be encrypted and securely distributed to each data center, where they can be utilized for local SSL/TLS termination. Geo Key Manager allows you to choose where you want to store your private keys. Geo Key Manager was restricted to the US, EU, and high-security data centers, but with the new version of Geo Key Manager, available in [Closed Beta](https://blog.cloudflare.com/configurable-and-scalable-geo-key-manager-closed-beta/), we allow you to create `allowlists` and `blocklists` of countries in which your private keys will be stored. That means that you will be able to create specifications, for instance store your private keys only in Australia or store your private keys in the EU and in the UK.
```
Comment (marciocloudflare) [inline content/data-localization/key-management/geo-key-manager.md:9]: There's something missing in the sentence `That means that you will be able to create specifications, for instance store your private keys...`. Specifications of what?

File: content/data-localization/key-management/geo-key-manager.md [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+
+For more information, refer to [Geo Key Manager docs](/ssl/edge-certificates/custom-certificates#geo-key-manager-private-key-restriction).
```
Comment (marciocloudflare) [inline content/data-localization/key-management/geo-key-manager.md]:
```suggestion
For more information, refer to [Geo Key Manager documentation](/ssl/edge-certificates/custom-certificates#geo-key-manager-private-key-restriction).
```

File: content/data-localization/key-management/keyless-ssl.md [hunk 14]
```diff
@@ -0,0 +1,11 @@
+---
+pcx_content_type: concept
+title: Keyless SSL
+weight: 3
+---
+
+# Keyless SSL
+
+The standard Cloudflare SSL service requires a customer to share their site’s SSL key. However, for some customers there are policy or technical obstacles preventing them from sharing their site’s SSL key. Keyless SSL allows you to use Cloudflare’s SSL service while retaining on-premise custody of your private keys.
```
Comment (marciocloudflare) [inline content/data-localization/key-management/keyless-ssl.md]:
```suggestion
The standard Cloudflare SSL service requires a customer to share the SSL key for their site. However, for some customers there are policy or technical obstacles preventing them from sharing the SSL key for their site. Keyless SSL allows you to use Cloudflare’s SSL service while retaining on-premise custody of your private keys.
```

File: content/data-localization/key-management/keyless-ssl.md [hunk 15]
```diff
@@ -0,0 +1,11 @@
...
+
+For more information, refer to [Keyless SSL docs](/ssl/keyless-ssl/).
```
Comment (marciocloudflare) [inline content/data-localization/key-management/keyless-ssl.md]:
```suggestion
For more information, refer to the [Keyless SSL documentation](/ssl/keyless-ssl/).
```

File: content/data-localization/regional-services/_index.md [hunk 16]
```diff
@@ -0,0 +1,15 @@
...
+pcx_content_type: concept
+title: Regional Services
+weight: 2
+---
+
+# Regional Services
+
+Regional Services gives you the ability to accommodate regional restrictions by choosing which subset of data centers decrypt and service HTTPS traffic.
+
+Regional Services proceeds and processes traffic within certain regions for customers who have to meet regional compliance or have preferences for maintaining regional control over their data. Examples of use cases could be a customer that needs to accommodate regional restrictions like GDPR (General Data Protection Regulation), or customers that are bound by agreement with their own customers that include geographic restrictions on data flows or data processing.
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/_index.md:11]: I'm not sure of what `proceeds` means in this context.

File: content/data-localization/regional-services/get-started.md [hunk 17]
```diff
@@ -0,0 +1,234 @@
+---
+title: Get started
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md:2]: I don't think this page should be Get started (or of content type get-started) bc it's a sub-page. Up until now, get started pages are main pages. Maybe change to `Enable Regional Services`?

File: content/data-localization/regional-services/get-started.md [hunk 18]
```diff
@@ -0,0 +1,234 @@
...
+pcx_content_type: get-started
+weight: 1
+layout: list
+---
+
+## Get Started
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
# Get Started
```

Comment (marciocloudflare) [reply content/data-localization/regional-services/get-started.md]: Even if you end up not changing this from "Get started" to something else, all titles and subtitles are wrong bc you began the page with an H2 instead of an H1.

File: content/data-localization/regional-services/get-started.md [hunk 19]
```diff
@@ -0,0 +1,234 @@
...
+
+You can use Regional Services through the dashboard or via API.
+
+### Regional Services in the dashboard
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
## Regional Services in the dashboard
```

File: content/data-localization/regional-services/get-started.md [hunk 20]
```diff
@@ -0,0 +1,234 @@
...
+
+To use Regional Services, you need to first create a DNS record in the dashboard:
+
+1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/) and select an account and domain.
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/), and select an account and domain.
```

File: content/data-localization/regional-services/get-started.md [hunk 21]
```diff
@@ -0,0 +1,234 @@
...
+3. Follow these steps to [create a DNS record](/dns/manage-dns-records/how-to/create-dns-records/).
+4. From the Region dropdown, select the region you would like to use on your domain. The complete list of available regions is the following:
+
+- Australia
+- Canada
+- European Union
+- India
+- Japan
+- United States
+- FedRAMP Compliant
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md]:
```suggestion
4. From the **Region** dropdown, select the region you would like to use on your domain. The complete list of available regions is the following:

    - Australia
    - Canada
    - European Union
    - India
    - Japan
    - United States
    - FedRAMP Compliant
```

Comment (marciocloudflare) [reply content/data-localization/regional-services/get-started.md]: I added 4 spaces so the list is indented (it should be rendered as belonging to step 4, not an independent list).

File: content/data-localization/regional-services/get-started.md [hunk 22]
```diff
@@ -0,0 +1,234 @@
...
+
+### Regional Services via API
+
+You can also use Regional Services via API. These are some examples of API requests.
```
Comment (marciocloudflare) [inline content/data-localization/regional-services/get-started.md:31]: Do we have API docs to link this to?

File: content/data-localization/metadata-boundary/_index.md [hunk 23]
```diff
@@ -0,0 +1,12 @@
+---
+pcx_content_type: concept
+title: Customer Metadata Boundary
+weight: 3
+---
+
+# Customer Metadata Boundary
+
+
+The Customer Metadata Boundary (CMB), as part of the Data Localization Suite, ensures that any customer end user traffic metadata that identifies that customer (that is, contains that customer's Account ID) stays in the EU or in the US, depending on the region the customer selects. That means, for example, that if a customer selects the EU Customer Metadata Boundary, metadata will only be sent to our core data center in the EU and not our core data center in the US.
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/_index.md]:
```suggestion
As part of the Data Localization Suite, the Customer Metadata Boundary (CMB) ensures that any traffic metadata that identifies a customer’s end user (that is, contains that customer's Account ID) stays in the EU or in the US, depending on the region the customer selects. That means, for example, that if a customer selects the EU Customer Metadata Boundary, metadata will only be sent to our core data center in the EU and not our core data center in the US.
```

Comment (marciocloudflare) [reply content/data-localization/metadata-boundary/_index.md]: Could you check if this was the meaning of that 1st sentence? To be honest I found it hard to follow the idea behind `any customer end user traffic metadata that identifies that customer`

File: content/data-localization/metadata-boundary/get-started.md [hunk 24]
```diff
@@ -0,0 +1,71 @@
+---
+title: Get started
+pcx_content_type: get-started
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md:3]: I don't think this should be a get started page. Please see my comment on Regional Services > Get started page.

File: content/data-localization/metadata-boundary/get-started.md [hunk 25]
```diff
@@ -0,0 +1,71 @@
...
+
+These are some examples of API requests. 
+
+<details>
+<summary>Get current regions</summary>
+<div>
+
+Here is an example request using cURL to get current regions (if any):
+
+```json
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md]:
```suggestion
```bash
```

File: content/data-localization/metadata-boundary/get-started.md [hunk 26]
```diff
@@ -0,0 +1,71 @@
...
+</div>
+</details>
+
+<details>
+<summary>Setting regions</summary>
+<div>
+
+Here is an example request using cURL to set regions:
+
+```json
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md]:
```suggestion
```bash
```

File: content/data-localization/metadata-boundary/get-started.md [hunk 27]
```diff
@@ -0,0 +1,71 @@
...
+<summary>Get current regions</summary>
+<div>
+
+Here is an example request using cURL to get current regions (if any):
+
+```json
+curl -s -D "/dev/stderr" https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/logs/control/cmb/config -X GET \
+-H "X-Auth-Email: <EMAIL>" \
+-H "X-Auth-Key: <KEY>" \
+| jq '.'
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md:27]: Please confirm this formatting (indenting and the place where the several sentences appear). It seems weird and I think it might be wrong

File: content/data-localization/metadata-boundary/get-started.md [hunk 28]
```diff
@@ -0,0 +1,71 @@
...
+
+```json
+curl -s -D "/dev/stderr" https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/logs/control/cmb/config -X POST -d '
+{
+    "regions": "eu"
+}
+' \
+-H "X-Auth-Email: <EMAIL>" \
+-H "X-Auth-Key: <KEY>" \
+| jq '.'
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md:47]: Please confirm this formatting (indenting and the place where the several sentences appear). It seems weird and I think it might be wrong

File: content/data-localization/metadata-boundary/get-started.md [hunk 29]
```diff
@@ -0,0 +1,71 @@
...
+</div>
+</details>
+
+<details>
+<summary>Delete regions</summary>
+<div>
+
+Here is an example request using cURL to delete regions:
+
+```json
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md]:
```suggestion
```bash
```

File: content/data-localization/metadata-boundary/get-started.md [hunk 30]
```diff
@@ -0,0 +1,71 @@
...
+curl -s -D "/dev/stderr" https://api.cloudflare.com/client/v4/accounts/<ACCOUNT_ID>/logs/control/cmb/config -X DELETE \
+-H "X-Auth-Email: <EMAIL>" \
+-H "X-Auth-Key: <KEY>" \
+| jq '.'
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/get-started.md:67]: Please confirm this formatting (indenting and the place where the several sentences appear). It seems weird and I think it might be wrong

File: content/data-localization/metadata-boundary/faq.md [hunk 31]
```diff
@@ -0,0 +1,30 @@
+---
+title: FAQ
+pcx_content_type: FAQ
+weight: 2
+---
+
+# FAQ
+
+## What data is covered by the Customer Metadata Boundary?
+Nearly all end user metadata is covered by the Customer Metadata Boundary. This includes all of the end user data for which Cloudflare is a processor, as defined in the Cloudflare Privacy Policy. Cloudflare is a data processor of Customer Logs, which are defined as end user logs that we make available to our customers via the dashboard or other online interfaces. End users are those who access or use our customers’ domains, networks, websites, application programming interfaces, and applications.
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/faq.md]:
```suggestion

Nearly all end user metadata is covered by the Customer Metadata Boundary. This includes all of the end user data for which Cloudflare is a processor, as defined in the [Cloudflare Privacy Policy](https://www.cloudflare.com/privacypolicy/). Cloudflare is a data processor of Customer Logs, which are defined as end user logs that we make available to our customers via the dashboard or other online interfaces. End users are those who access or use our customers’ domains, networks, websites, application programming interfaces, and applications.
```

File: content/data-localization/metadata-boundary/faq.md [hunk 32]
```diff
@@ -0,0 +1,30 @@
...
+
+Specific examples of this data include all of the analytics in our dashboard and APIs on requests, responses, and security products associated and all of the logs received through Logpush.
+
+## What data is not covered by the Customer Metadata Boundary?
+Some of the data for which Cloudflare is a controller, as defined in the Cloudflare Privacy Policy.
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/faq.md]:
```suggestion

Some of the data for which Cloudflare is a controller, as defined in the [Cloudflare Privacy Policy](https://www.cloudflare.com/privacypolicy/).
```

File: content/data-localization/metadata-boundary/faq.md [hunk 33]
```diff
@@ -0,0 +1,30 @@
...
+
+Some examples:
+Customer account data (for example, name and billing information)
+Customer configuration data (for example, the content of Firewall Rules)
+Metadata that is “operational” in nature —  data needed for Cloudflare to properly operate our network. This includes metadata such as:
+System data generated for debugging (for example, application logs from internal systems, core dumps)
+Networking flow data (for example, sFlow from our routers), including data on DDoS attacks
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/faq.md]:
```suggestion
Some examples:

- Customer account data (for example, name and billing information).
- Customer configuration data (for example, the content of Firewall Rules).
- Metadata that is “operational” in nature —  data needed for Cloudflare to properly operate our network. This includes metadata such as:
    - System data generated for debugging (for example, application logs from internal systems, core dumps).
    - Networking flow data (for example, sFlow from our routers), including data on DDoS attacks.
```

Comment (marciocloudflare) [reply content/data-localization/metadata-boundary/faq.md]: As you had each sentence in a new paragraph, I changed this to an unordered list bc without it, this paragraph is rendered as a big block of text.

File: content/data-localization/metadata-boundary/faq.md [hunk 34]
```diff
@@ -0,0 +1,30 @@
...
+
+## Who can use the Customer Metadata Boundary?
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/faq.md:27]:
```suggestion
## Who can use the Customer Metadata Boundary?

```

File: content/data-localization/metadata-boundary/faq.md [hunk 35]
```diff
@@ -0,0 +1,30 @@
...
+Currently, this is available for Enterprise customers as part of the Data Localization Suite.
+
+The Customer Metadata Boundary is for customers who want to limit personal data transfer outside the EU or the US (depending on the customer’s selected region). These customers should already be using Regional Services, which ensures that traffic content is only ever decrypted within the geographic region specified by the customer.
+
+## What are the analytics products available for Metadata Boundary?
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/faq.md]:
```suggestion
## What are the analytics products available for Metadata Boundary?

```

File: content/data-localization/metadata-boundary/_index.md [hunk 36]
```diff
@@ -0,0 +1,12 @@
+---
+pcx_content_type: concept
+title: Customer Metadata Boundary
+weight: 3
+---
+
+# Customer Metadata Boundary
+
```
Comment (marciocloudflare) [inline content/data-localization/metadata-boundary/_index.md:8]:
```suggestion
# Customer Metadata Boundary
```

Comment (marciocloudflare) [reply content/data-localization/metadata-boundary/_index.md:8]: Removed one extra line

**Review by marciocloudflare — APPROVED** [review]
